packages/eslint-config-edx/.eslintrc.json

```diff
@@ -6,14 +6,19 @@
         "jasmine": true,
         "jquery": true
     },
-    "extends": "airbnb-base",
+    "extends": "airbnb",
     "parserOptions": {
-        "ecmaVersion": 8
+        "ecmaVersion": 8,
+        "ecmaFeatures": {
+          "jsx": true
+        }
     },
     "plugins": [
         "dollar-sign"
     ],
     "rules": {
-        "dollar-sign/dollar-sign": ["error", "ignoreProperties"]
+        "dollar-sign/dollar-sign": ["error", "ignoreProperties"],
+        "react/no-array-index-key": "off",
```

Comment:
I'm not 100% sure about this one. I've found a few articles that suggest that it is dangerous:

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Why do you want to disable this?